### ISSUE TITLE
Tab key should work as tabulator in Define keyboard shortcut dialog

### ISSUE THREAD

**Issue by DmitryArefiev** [musescore/MuseScore#14719]
**Describe the bug**
Tab key should work as tabulator in Define keyboard shortcut dialog. This is **critical for blind users** because they can't navigate through dialog and select Save

https://user-images.githubusercontent.com/90187801/203761948-bd57f410-100d-4b80-a525-d1ebbbe6b59d.mp4

**Expected behavior**

**Screenshots**
If applicable, add screenshots to help explain your problem

**Platform information**
macOS, Windows, Linux

**Additional info**
See comment from Marc https://github.com/musescore/MuseScore/issues/14719#issuecomment-1405747906
.

**Comment by MarcSabatella** [comment]
A blind user is also seems to be the case that the Save button doesn't read even when forcing focus by hovering the mouse over it (something I guess you can configure on some systems; I don't know how that works myself).

So right now, between Enter not activating the Save button and Tab not navigating there, it does seem there is no way for blind users to define shortcuts currently.

**Comment by bkunda** [comment]
I'm giving this one a little bump in priority on our project roadmap. 
It strikes me that being unable to define keyboard shortcuts would represent a major limitation to effectively using the app without a mouse.

**Comment by DmitryArefiev** [comment]
This was fixed in #22036 for #16157

Closing
